galactic/distribution.yaml

```diff
@@ -2619,7 +2619,7 @@ repositories:
       tags:
         release: release/galactic/{package}/{version}
       url: https://github.com/ros2-gbp/rmf_cmake_uncrustify-release.git
-      version: 1.2.0-1
+      version: 1.2.0-2
```

Comment:
@marcoag hmm, why re-release ? I can't seem to find changes upstream.

Reply:
Thanks for the check. I believe there's a maintainer added on the `package.xml` that we wanted to add for notifications purposes.

Reply:
I see. And the source tag was recreated I presume. That's a tiny bit unusual but OK.

Reply:
Just trying to understand the process: shouldn't the source tag be recreated as the `package.xml` was modified?

Reply:
I think that standard practice would be to bump the version number of the package to avoid force-pushing a tag.

Reply:
I see, I thought since it's not a feature nor a fix I would keep the version number unchanged, but good to know for next time. Thanks!